cdap-storage-ext-spanner/src/main/java/io/cdap/cdap/storage/spanner/SpannerStructuredTable.java

```diff
@@ -367,6 +368,28 @@ public void deleteAll(Range range) throws InvalidFieldException {
     transactionContext.executeUpdate(builder.build());
   }
 
+  @Override
+  public void updateAll(Range keyRange, Collection<Field<?>> fields) throws InvalidFieldException, IOException {
+    fieldValidator.validatePrimaryKeys(keyRange.getBegin(), true);
+    fieldValidator.validatePrimaryKeys(keyRange.getEnd(), true);
+
+    Map<String, Value> parameters = new HashMap<>();
```

Comment:
Did we bind these `parameters` for the WHERE clause? Something like `parameters.forEach((name, value) -> builder.bind(name).to(value));`

Reply:
this makes more sense after debugging. Good catch, thanks!